<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResumeReadWindow</class>
 <widget class="QDialog" name="ResumeReadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Дані резюме</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="11" column="0">
       <widget class="QLabel" name="l_company">
        <property name="text">
         <string>Клаліфікація</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="l_city">
        <property name="text">
         <string>Хоббі</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="le_hobbies"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_salary">
        <property name="text">
         <string>Комунікативні навики</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QTextEdit" name="te_about_myself">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="l_email">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_education">
        <property name="text">
         <string>Професійні навики</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_pip"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_language"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_soft_skills"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_experience">
        <property name="text">
         <string>Знання мов</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="l_description">
        <property name="text">
         <string>Про себе</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="l_phone">
        <property name="text">
         <string>Телефон</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="l_location">
        <property name="text">
         <string>Skype</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="l_title">
        <property name="text">
         <string>Піп</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_hard_skills"/>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="le_phone"/>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="le_email"/>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="le_skype"/>
      </item>
      <item row="11" column="2">
       <widget class="QLineEdit" name="le_qualification"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
